crates/sui-test-validator/src/main.rs

```diff
@@ -1,263 +1,28 @@
-        .route("/v1/status/:task_id", get(request_status))
-        .layer(
-            ServiceBuilder::new()
-                .layer(cors)
-                .layer(Extension(app_state))
-                .into_inner(),
-        );
-
-    let addr = SocketAddr::from(([0, 0, 0, 0], port));
-
-    println!("Faucet URL: http://{}", addr);
-
-    axum::Server::bind(&addr)
-        .serve(app.into_make_service())
-        .await?;
-
-    Ok(())
-}
-
-/// basic handler that responds with a static string
-async fn health() -> &'static str {
-    "OK"
-}
-
-async fn faucet_request(
-    Extension(state): Extension<Arc<AppState>>,
-    Json(payload): Json<FaucetRequest>,
-) -> impl IntoResponse {
-    let result = match payload {
-        FaucetRequest::FixedAmountRequest(FixedAmountRequest { recipient }) => {
-            state.faucet.request_sui_coins(recipient).await
-        }
-        _ => {
-            return (
-                StatusCode::BAD_REQUEST,
-                Json(FaucetResponse::from(FaucetError::Internal(
-                    "Input Error.".to_string(),
-                ))),
-            )
-        }
-    };
-
-    if !result.transferred_gas_objects.is_empty() {
-        (StatusCode::CREATED, Json(result))
-    } else {
-        (StatusCode::INTERNAL_SERVER_ERROR, Json(result))
-    }
-}
-
-async fn faucet_batch_request(
-    Extension(state): Extension<Arc<AppState>>,
-    Json(payload): Json<FaucetRequest>,
-) -> impl IntoResponse {
-    let result = match payload {
-        FaucetRequest::FixedAmountRequest(FixedAmountRequest { recipient }) => {
-            state.faucet.batch_request_sui_coins(recipient).await
-        }
-        _ => {
-            return (
-                StatusCode::BAD_REQUEST,
-                Json(BatchFaucetResponse::from(FaucetError::Internal(
-                    "Input Error.".to_string(),
-                ))),
-            )
-        }
-    };
-    if result.task.is_some() {
-        (StatusCode::CREATED, Json(result))
-    } else {
-        (StatusCode::INTERNAL_SERVER_ERROR, Json(result))
-    }
-}
-
-async fn request_status(
-    Extension(state): Extension<Arc<AppState>>,
-    Path(id): Path<String>,
-) -> impl IntoResponse {
-    match Uuid::parse_str(&id) {
-        Ok(task_id) => {
-            let status = state.faucet.get_batch_send_status(task_id).await;
-            (StatusCode::CREATED, Json(status))
-        }
-        Err(e) => (
-            StatusCode::INTERNAL_SERVER_ERROR,
-            Json(BatchStatusFaucetResponse::from(FaucetError::Internal(
-                e.to_string(),
-            ))),
-        ),
-    }
+fn main() {
+    let info = "sui-test-validator binary has been deprecated in favor of sui start, which is a \
+                more powerful command that allows you to start the local network with more options. \
+There are three options that you can use to replace `sui-test-validator` binary:
+  * use `sui-pq` binary from a release archive from the GitHub releases page
```

Comment:
```suggestion
  * use `sui-pg` binary from a release archive from the GitHub releases page
```

Reply:
It may be worth clarifying here that you only need to use `sui-pg` (or the other options here) if you want to use indexer or GraphQL, and otherwise you can just switch straight to `sui start`.